eslint/flat/import.js

```diff
@@ -0,0 +1,10 @@
+const importPlugin = require('eslint-plugin-import');
+
+/** @type {improt('eslint').Linter.Config} */
```

Comment:
You have a typo here. `import` instead of `improt`

Reply:
Thank you!